### DISCUSSION TITLE
Can not import pixi.JS as node package

### DISCUSSION THREAD

**Discussion by LeonariX58** [pixijs/pixijs#8300] [Q&A]
Error:
       Uncaught TypeError: Failed to resolve module specifier "pixi.js". Relative references must start with either "/", "./", or "../".
       
Code:
```
        <!doctype html>
      <html>
        <head>
        </head>
        <body>
            <div id="display"></div>
            <script type="module">
              import * as PIXI from 'pixi.js'
            </script>
              <!--<script type="module" src="main.js"></script>-->
      
        </body>
      </html> 
```
     
![image](https://user-images.githubusercontent.com/75447012/164585311-baa0b0c8-81b7-40a2-bf57-f97d7fa0d03a.png)

**Comment by miltoncandelero** [comment]
Are you using some sort of bundler like webpack?
May I suggest a template project like https://github.com/miltoncandelero/pixi-hotwire ?

**Comment by bigtimebuddy** [comment]
If you want to import as a module directly in the browser, check out this example:

https://github.com/bigtimebuddy/pixi.js-browser-module-example

The pixi.mjs file is available in the package from npm, from a CDN, or direct download from the releases section.
